openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx

```diff
@@ -110,13 +110,25 @@ const FacetFilter: FunctionComponent<FacetProp> = ({
     }
   };
 
+  const getTagsWithLabel = (sortedTags: Array<Bucket>) => {
```

Comment:
Can we move this to util and add comments on what this method will be doing?

Reply:
Updated!